view/next-project/src/components/sponsoractivities/SponsorActivitiesAddModal.tsx

```diff
@@ -253,41 +267,25 @@ export default function SponsorActivitiesAddModal(props: Props) {
           </option>
         </Select>
       </div>
+      <p className='text-black-600 text-center'>広告データurl</p>
+      <div className={clsx('col-span-4 grid w-full')}>
+        <Input value={data.url} onChange={formDataHandler('url')} />
+      </div>
       <p className='text-black-600'>デザイン作成</p>
       <div className='col-span-4 flex w-full justify-around'>
-        <div className='flex gap-3'>
-          <input
-            type='radio'
-            id='student'
-            name='design'
-            value='学生が作成'
-            checked={design === '学生が作成'}
-            onChange={(e) => setDesign(e.target.value)}
-          />
-          <label htmlFor='student'>学生が作成</label>
-        </div>
-        <div className='flex gap-3'>
-          <input
-            type='radio'
-            id='company'
-            name='design'
-            value='企業が作成'
-            checked={design === '企業が作成'}
-            onChange={(e) => setDesign(e.target.value)}
-          />
-          <label htmlFor='company'>企業が作成</label>
-        </div>
-        <div className='flex gap-3'>
-          <input
-            type='radio'
-            id='lastYear'
-            name='design'
-            value='去年のものを使用'
-            checked={design === '去年のものを使用'}
-            onChange={(e) => setDesign(e.target.value)}
-          />
-          <label htmlFor='lastYear'>去年のものを使用</label>
-        </div>
+        {DESIGNER_VALUES.map((designer) => (
+            <div className='flex gap-3'>
```

Comment:
すまん、レビューでkeyの指定を書いてなかった！
mapで繰り返し表示したところにはkeyを書いておきたい
参考：https://ja.legacy.reactjs.org/docs/lists-and-keys.html

```suggestion
            <div className='flex gap-3' key={designer.id}>
```